README

```diff
@@ -0,0 +1,88 @@
+NAME
+    App::Config - The great new App::Config!
+
+VERSION
+    Version 0.01
+
+SYNOPSIS
+        my $app_config = App::Config->new;
+
+DESCRIPTION
+    This module parses configuration files and provides interface to access
+    configuration information.
+
+FILE FORMAT
+    The configuration file is a YAML file. Here is an example:
+
+        system:
+          description: "Various parameters determining core application functionality"
+          isa: section
+          contains:
+            email:
+              description: "Dummy email address to indicate email sent from our system"
+              isa: Str
+              default: "dummy@binary.com"
+              global: 1
+            admins:
+              description: "Are we on Production?"
+              isa: ArrayRef
+              default: []
+
+    Every atribute is very intuitive. If an item is global, you can change
+    its value and the value will be stored into chronicle database by
+    calling the method "save_dynamic".
+
+SUBROUTINES/METHODS
+  definition_yml
+    The YAML file that store the configuration
+
+  chronicle_reader
```

Comment:
mention that this relates to Data::Chronicle. It isn't obvious to someone new to this module.